backend/dal/Services/LookupService.cs

```diff
@@ -157,6 +157,11 @@ public IEnumerable<PimsInsuranceType> GetInsuranceTypes()
         {
             return this.Context.PimsInsuranceTypes.AsNoTracking().OrderBy(a => a.InsuranceTypeCode).ToArray();
         }
+
+        public IEnumerable<PimsPropertyImprovementType> GetPropertyImprovementTypes()
+        {
+            return this.Context.PimsPropertyImprovementTypes.AsNoTracking().OrderBy(a => a.PropertyImprovementTypeCode).ToArray();
```

Comment:
Shouldn't we exclude those types that are disabled here?

Reply:
property improvement table doesn't have a disabled column

Reply:
yes, will fix.

Reply:
this is for the improvement_type, right? the IS17 schema shows it has disabled...

![image](https://user-images.githubusercontent.com/24322224/148627497-8149e919-f2d3-4859-a2b6-e7efad43534c.png)

Reply:
And the entity has it (at least my copy)
![image](https://user-images.githubusercontent.com/24322224/148627529-d4a9559f-1b78-47fc-8589-b6e46b10c875.png)

Reply:
actually, I take this back. The front end handles filtering disabled lookups via the lookup code helper, and I think there is value for the api to return disabled codes otherwise the FE has no way of interacting with them at all.

Reply:
Fine but we will need to decide on a common strategy for all lookup codes. We were discussing to do things server side on my PR, now we move it to the frontend. Seems fine but need to settle on one way

Reply:
Conceptually not sure why we would need the frontend to know about "disabled" lookup codes. To me disabled kinda equals "deleted from the app" but maybe I'm thinking about it wrong

Reply:
We already support the concept of disabled contacts, so for me it isn't a stretch that the same logic could apply to other values.